src/transitions/Grow.js

```diff
@@ -0,0 +1,172 @@
+// @flow
+
+import React, { Component } from 'react';
+import type { Element } from 'react';
+import withTheme from '../styles/withTheme';
+import Transition from '../internal/Transition';
+import type { TransitionCallback } from '../internal/Transition';
+
+export function getScale(value: number) {
+  return `scale(${value}, ${value ** 2})`;
+}
+
+type DefaultProps = {
+  theme: Object,
```

Comment:
It has to be required somewhere, and it is not required outside this component.  If it is not required outside the component, we cannot require it in `export type Props`, as it would prohibit using the type externally to compose components.